app/view/components/StyledText.js

```diff
@@ -21,4 +21,4 @@ const StyledText = (props) => {
   );
 };
 
-export default StyledText;
+export default React.memo(StyledText);
```

Comment:
I doubt this `React.memo` help us to have an optimized application. but it is negotiable. I guess it is better to have a conversation on the **Telegram** group or in the **Issues** tab. but now it is time to accepting the PR.